hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfigHolder.java

```diff
@@ -103,6 +103,11 @@ public class HiveSyncConfigHolder {
       .key("hoodie.datasource.hive_sync.create_managed_table")
       .defaultValue(false)
       .withDocumentation("Whether to sync the table as managed table.");
+  public static final ConfigProperty<Boolean> HIVE_OMIT_METADATA_FIELDS = ConfigProperty
+      .key("hoodie.datasource.hive_sync.omit_metadata_fields")
+      .defaultValue(false)
+      .sinceVersion("0.12.1")
```

Comment:
We don't introduce new configs in minor releases.   This should be `0.13.0` which is the next major version.